<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumberConverter</class>
 <widget class="QWidget" name="NumberConverter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>116</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="labelBin">
     <property name="text">
      <string>Bin</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelDec">
     <property name="text">
      <string>Dec</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEditBin"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelHex">
     <property name="text">
      <string>Hex</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEditOct"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelOct">
     <property name="text">
      <string>Oct</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditDec"/>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditHex"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
